packages/fx-core/tests/component/driver/aad/create.test.ts

```diff
@@ -597,9 +597,6 @@ describe("aadAppCreate", async () => {
     expect(endTelemetry.properties.success).to.equal("no");
     expect(endTelemetry.properties["error-code"]).to.equal("aadAppCreate.HttpClientError");
     expect(endTelemetry.properties["error-type"]).to.equal("user");
-    // expect(endTelemetry.properties["error-message"]).to.equal(
```

Comment:
I saw you commented this validation in https://github.com/OfficeDev/teams-toolkit/commit/69d25b5448d01d1f4b0e0965c524c3bf97b76acf#diff-0b879376e09c63caffd785a473e6a10328740fb818f82b76a023114a17917589.

Can you help recall why this check needs to be removed and add it back if possible? No need to address in this PR. We can update in another PR to avoid blocking your PR.

Reply:
Because the 'err-message' property will be processed and not equal to the original one